### PR TITLE
Implement share button

### LINKED ISSUE #54: Share Button
### Is your feature request related to a problem? Please describe.

I want to quickly share my results and it's tedious to write it myself

### Describe the solution you'd like

I would like to see a button that copies my results to the clipboard

### Additional context

Many other sites have this, including progle

### PR DESCRIPTION
**Pull request by ricardoboss**
This PR is based on #68. So that one needs to be merged first.

---

Fixes #54

### CODE REVIEW

**Comment by ricardoboss** [comment]
This game: 
image

Results in this output:

🟨🟩🟥🟩🟨🟨🔽🔼
🟨🟩🟩🟩🟨🟨🔼🔽
🟨🟩🟩🟥🟥🟨🔼🔼
🟨🟩🟥🟥🟥🟨🔼🔼
🟩🟩🟩🟩🟩🟩🟩🟩

File: Prolangle/wwwroot/js/app.js [hunk 1]
```diff
@@ -9,3 +9,7 @@ function getBoundingBox(id) {
         Height: rect.height
     };
 }
+
+async function writeTextToClipboard(text) {
+    await navigator.clipboard.writeText(text);
```
Comment (chucker) [inline Prolangle/wwwroot/js/app.js:14]: I vaguely recall reading that this won't work in all browsers.

Comment (ricardoboss) [reply Prolangle/wwwroot/js/app.js:14]: image

It looks like it should work on all, but I can't test this on iOS.

Comment (chucker) [reply Prolangle/wwwroot/js/app.js:14]: Ah, it's probably the "must be called within user gesture event handlers" bit. Which you do seem to do.

**Review by chucker — APPROVED** [review]

**Comment by chucker** [comment]
Two suggestions (not sure whether to turn this into a draft again; I'm OK with shipping this as a first version):

* limit the lines, but signal the limit, e.g. through a vertical ellipsis (not the prettiest choice):

🟨🟩🟥🟩🟨🟨🔽🔼
🟨🟩🟩🟩🟨🟨🔼🔽
⋮
🟩🟩🟩🟩🟩🟩🟩🟩

* instead of the clipboard, how about `navigator.share()`?

**Review by chucker — APPROVED** [review]

**Comment by chucker** [comment]
I get

```
crit: Microsoft.AspNetCore.Components.WebAssembly.Rendering.WebAssemblyRenderer[100]
      Unhandled exception rendering component: Abort due to cancellation of share.
      
Microsoft.JSInterop.JSException: Abort due to cancellation of share.

   at Microsoft.JSInterop.JSRuntime.<InvokeAsync>d__16`1[[System.String, System.Private.CoreLib, Version=8.0.0.0, Culture=neutral, PublicKeyToken=7cec85d7bea7798e]].MoveNext()
   at Prolangle.Components.ShareMetadatumComparisonButton.OnClicked()
   at Microsoft.AspNetCore.Components.ComponentBase.CallStateHasChangedOnAsyncCompletion(Task task)
   at Microsoft.AspNetCore.Components.RenderTree.Renderer.GetErrorHandledTask(Task , ComponentState )
```

when cancelling.

**Pull request merged**